vtex/admin-promotions
# Fix search for archived promotions

#### What is the purpose of this pull request?
As title says.
Depends on https://github.com/vtex/admin-promotions-graphql/pull/57

#### How should this be manually tested?
Search for "gratis". It should find at least 3 promotions, some with special chars
http://vlauxbeta--pricingqa.myvtex.com/admin/promotions/archived

Also works for active promos:
Try searching for "promocao"
http://vlauxbeta--pricingqa.myvtex.com/admin/promotions/

#### Types of changes
